test/src/main/java/org/apache/accumulo/harness/SharedMiniClusterBase.java

```diff
@@ -148,14 +154,11 @@ private static String getTestClassName() {
    */
   public static synchronized void stopMiniCluster() {
     if (STOP_DISABLED.get()) {
-      // If stop is disabled, then we are likely running a
-      // test class that is part of a larger suite. We don't
-      // want to shut down the cluster, but we should clean
-      // up any tables that were created, but not deleted,
-      // by the test class. This will prevent issues with
-      // subsequent tests that count objects or initiate
-      // compactions and wait for them, but some other table
-      // from a prior test is compacting.
+      // If stop is disabled, then we are likely running a test class that is part of a larger
```

Comment:
I wonder if this should also reset the system config (and any table config affecting accumulo.\* tables)?